### DISCUSSION TITLE
"Copy Playlist" - maps fail to copy, date error when creating playlist from a copy

### DISCUSSION THREAD

**Discussion by itotallygetthat** [ppy/osu#17154] [Q&A] [answered]
2022.306.0
[network.log](https://github.com/ppy/osu/files/8202739/network.log)
[runtime.log](https://github.com/ppy/osu/files/8202741/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8202742/updater.log)
[database.log](https://github.com/ppy/osu/files/8202743/database.log)
[performance.log](https://github.com/ppy/osu/files/8202744/performance.log)

Yes, this is probably technically 2 separate issues, but they seem possibly related.

1. If you click "Copy Playlist" without having just had that same Playlist open, none of the maps from the original Playlist are copied to the new Playlist.  (To replicate, open one playlist, close it, then copy a different playlist.  Or just copy any playlist without opening it first.)
2. More importantly, when clicking "Create" on a copied playlist to actually upload your Playlist (whether the maps copied correctly or not), you get an error in red text like '"ends_at" must be at least 30 minutes after started_at'.  And the Playlist fails to create.  Changing the Playlist duration does not affect whether this error occurs.   (Creating a new Playlist normally without being a copy still functions correctly.)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/17186

**Reply by itotallygetthat** [reply]
Part 2 is still an issue in .314

After clicking Copy Playlist, clicking the "Create" button to finalize the playlist gives a red text warning about "'ends_at' must be at least 30 minutes after 'starts_at'".  Playlist creation fails due to this error.

**Reply by peppy** [reply]
Remaining part is covered by https://github.com/ppy/osu/pull/17253.
